CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 ### Changed
 
 ### Fixed
+- Fixed warnings about keyword arguments in Ruby 2.7
+  - The official explanation of keyword arguments in Ruby 2.7 can be found [HERE](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/)
+([oieioi])
```

Comment:
Add this below all other contributors at bottom and I'll merge. It just makes formatting proper.

Reply:
@luke-hill Added.